assets/prototype/application/utilities/text/changeCase/index.test.ts

```diff
@@ -0,0 +1,45 @@
+import { lcFirst, ucFirst } from './index';
+
+describe('lcFirst', () => {
+	test('should lowercase first letter of a string', () => {
+		expect(lcFirst('Test')).toBe('test');
+	});
+
+	test(`if first character is already lowered it shouldn't change anything`, () => {
+		expect(lcFirst('test')).toBe('test');
+	});
+
+	test(`shouldn't do anything if we pass a number as a string`, () => {
+		expect(lcFirst('123')).toBe('123');
+	});
+
+	test(`shouldn't do anything if we pass null`, () => {
+		expect(lcFirst(null)).toBe(undefined);
+	});
+
+	test(`shouldn't do anything if we pass undefined`, () => {
+		expect(lcFirst(undefined)).toBe(undefined);
+	});
```

Comment:
It's a bit weird why TS let me pass these, since `lcFirst` argument is typed to be a string.

Reply:
ya but it's always good in tests to confirm how your functions respond when garbage values get passed to them